onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1128,6 +1128,16 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                "2D tensor with shape (max_sequence_length, head_size / 2).",
                "T",
                OpSchema::Optional)
+        .Input(9,
+               "custom_pos_ids",
```

Comment:
Can this be named as `position_ids` instead to be more generic? The input values will anyways be customized for your scenario.

```suggestion
               "position_ids",
```